_data/events_gdcr2019/atlanta.json

```diff
@@ -0,0 +1,29 @@
+{
+    "title": "GDCR 2019 in Atlanta",
+    "url": "https://www.coderetreatatl.com/",
+    "moderators": [
+      {
+        "name": "Mike Clement",
+        "url": "https://twitter.com/mdclement"
+      }
+    ],
+    "sponsors": [
+      {
+        "name": "Greater Sum",
+        "url": "https://twitter.com/thegreatersum"
+      }
+    ],
+    "date": {
```

Comment:
hi, thanks for adding your event! We'll be using the timezone of all events again to figure out how to distribute the video chat rooms, so i'd appreciate if you would translate the start and end time into local times with the correct timezone suffix

Reply:
updated the dates with time zone info